example_test.go

```diff
@@ -0,0 +1,322 @@
+package qt_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/go-quicktest/qt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+func Example_Comment() {
+	runExampleTest(func(t testing.TB) {
+		a := 42
+		qt.Assert(t, qt.Equals(a, 42), qt.Commentf("no answer to life, the universe, and everything"))
```

Comment:
would be nice if this example actually failed, as then you show how the output looks like with the comment. might be tricky with filenames and such though.

Reply:
Yeah, I don't see a way to do that with the output containing a stack trace and all.